metrics/timer.go

```diff
@@ -5,7 +5,7 @@ import (
 	"time"
 )
 
-// Timers capture the duration and rate of events.
+// Timer methods capture the duration and rate of events.
```

Comment:
Please don't add `methods` all over, it's a simple interface.

Reply:
Thank you - updated.